[Rewrite] Function Wikis - #2 (#203)
NilPointer-Software/bdfd-wiki: src/bdscript/useChannel.md

# $useChannel
Redirects the bot's response message to a different channel.

> 📌 [`$useChannel`](./useChannel.md) cannot redirect the original slash command response.

## Syntax
```
$useChannel[Channel ID]
```

### Parameters

> - `Channel ID` : The ID of the channel to where it should redirect.

Reviewer: ```suggestion
- `Channel ID` : The ID of the channel to where the message will be redirected.
```
